formatMessage.go

```diff
@@ -75,10 +82,20 @@ func FormatMessage(emailReq EmailRequest, payload map[string](interface{}), deSe
 	payload["DEPublicationRequestsLink"] = deSettings.base + deSettings.admin + deSettings.apps
 	payload["DEPidRequestLink"] = deSettings.base + deSettings.admin + deSettings.doi
 
-	tmpl, err := template.ParseFiles("./templates/"+emailReq.Template+".tmpl", "./templates/header.tmpl", "./templates/footer.tmpl")
+	var isHtml bool = false
+	var tmpl Templater
+	var err error
+
+	if _, err = os.Stat("./templates/html/" + emailReq.Template + ".tmpl"); err == nil {
+		isHtml = true
+		tmpl, err = html.ParseFiles("./templates/html/"+emailReq.Template+".tmpl", "./templates/html/header.tmpl", "./templates/html/footer.tmpl")
+	} else if _, err = os.Stat("./templates/text/" + emailReq.Template + ".tmpl"); err == nil {
+		tmpl, err = text.ParseFiles("./templates/text/" + emailReq.Template + ".tmpl")
+	}
```

Comment:
This is pretty much the core logic. If I haven't screwed it up, it should use the `html/template` implementation and the `text/html` mime type when something exists in `templates/html`, and `text/template` and `text/plain` when one exists in `templates/text`. I haven't added a fall-through case for when neither exists, which I guess might be good, but anyway.

Reply:
LGTM so far 👍🏽